hw/top_earlgrey/data/ip/chip_kmac_testplan.hjson

```diff
@@ -68,6 +68,9 @@
       stage: V2
       si_stage: NA
       tests: ["chip_sw_kmac_app_rom"]
+      bazel: [
+        "//sw/device/tests:kmac_app_rom_test",
+      ]
```

Comment:
Do we want to merge the `chip_sw_kmac_app_rom` and `chip_sw_rom_ctrl_digests` testplan tests? If not, what's the best way to update them?

Reply:
It would be useful to have the opinion of someone doing DV, are those tests doing the same thing in DV? On real device, there isn't much to do since if there is a mismatch, it would not be running!

Reply:
@rswarbrick any opinion about this?

Reply:
They can definitely be merged, I think. The `rom_ctrl` test has a little bit of extra work (to check the digest that's reported in the `rom_ctrl` CSRs), but I can't see any other real difference.

I don't really have an opinion about which testplan the resulting testpoint should sit in :-)